### PR TITLE
Add ListEvents endpoint and table

### LINKED ISSUE #1569: Add error page to automation detail
I got a 500 on the request on the `kustomization?` page, but no error popped up. We should add that.

### LINKED ISSUE #1634: Implement tabs on details views
https://www.figma.com/file/IVHnM9iyeFWpd11evtY8ux/Weave-GitOps?node-id=6031%3A24061

### PR DESCRIPTION
**Pull request by jpellizzari**
Closes #1634 
Closes #1569

Adds events to automation and source detail pages

Screenshot from 2022-03-09 10-22-00

Edit: Also added stuff to change the proxy port and adjust the styling on the `UserSettings`.

### CODE REVIEW

File: ui/components/HashRouterTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      </div>
+    </div>
+  );
+}
+
+export default styled(HashRouterTabs).attrs({
+  className: HashRouterTabs.name,
+})`
+  .MuiTabs-root ${Link} .active-tab {
+    background: ${(props) => props.theme.colors.primary}19;
```
Comment (joshri) [inline ui/components/HashRouterTabs.tsx:106]: typo?

**Review by joshri — APPROVED** [review]
LGTM 🌞

File: ui/components/HashRouterTabs.tsx (hunk 1 above)
Comment (jpellizzari) [reply ui/components/HashRouterTabs.tsx:106]: nope, that is adding extra stuff to the hex value for opacity

Comment (joshri) [reply ui/components/HashRouterTabs.tsx:106]: got it nice

**Pull request merged**